charts/public-service/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+name: public-service
+description: Helm chart for deploying an instance of a public (HTTP, available from outside) service on a K8s cluster.
+version: 1.0.0
+sources:
+  - https://github.com/commercetools/commercetools-payone-integration.git
```

Comment:
why are the sources pointing to `commercetools-payone-integration` and `commercetools-paypal-plus-integration`? Isn't it this chart generic for any "public service" not just those two?

Reply:
aa1408c